partials/package-change.html

```diff
@@ -11,38 +11,51 @@
 				{{#if terms}}
 				<div class="ncf__package-change__terms">
 					{{#each terms}}
-					<div class="ncf__package-change__term">
+					<div class="ncf__package-change__term{{#if this.discount}} ncf__package-change__term__discounted{{/if}}">
```

Comment:
Can this be a modifier for a term 

```suggestion
					<div class="ncf__package-change__term{{#if this.discount}} ncf__package-change__term--discounted{{/if}}">
```